app/views/maintenance_tasks/tasks/_searchbar.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_tag(search_path, :method => "get") do %>
```

Comment:
Could this path just be `root_path` or `index`? No need to use a custom that redirects to `tasks#index`, you can just hit that route directly with the added query params.

Reply:
Good callout, I'll implement that change 🚀